samples/deploy/admission/webhook-registration.yaml

```diff
@@ -0,0 +1,40 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+apiVersion: admissionregistration.k8s.io/v1beta1
+kind: ValidatingWebhookConfiguration
+metadata:
+  name: apisix-validations
+  labels:
+    app: apisixroute-plugin-validator-webhook
+    kind: validating
+webhooks:
+  - name: apisixroute-plugin-validator-webhook
+    clientConfig:
+      service:
+        name: apisix-admission-server
+        namespace: ingress-apisix
+        port: 8443
+        path: "/validation/apisixRoute/plugin"
+      caBundle: ${CA_BUNDLE}
+    rules:
+      - operations: [ "CREATE", "UPDATE" ]
+        apiGroups: ["apisix.apache.org"]
+        apiVersions: ["*"]
+        resources: ["apisixroutes"]
+    timeoutSeconds: 30
+    failurePolicy: Ignore
```

Comment:
`failurePolicy` must be `Fail`

Reply:
Because this feature is just implemented, I'm afraid that it may be not stable enough. I prefer to set to `Fail` after I implement other webhooks (route/upstream/consumer) and add more test cases.